sdk/main/include/AccountCreateTransaction.h

```diff
@@ -44,7 +44,31 @@ class TransactionResponse;
 namespace Hedera
 {
 /**
- * Create a new Hedera account.
+ * The transaction used to create a new account. After the account is created, the AccountID for it is in the receipt.
+ * It can also be retrieved with a GetByKey query. Threshold values can be defined, and records are generated and stored
+ * for 25 hours for any transfer that exceeds the thresholds. This account is charged for each record generated, so the
+ * thresholds are useful for limiting record generation to happen only for large transactions.
+ *
+ * The mPublicKey field is the key used to sign transactions for this account. If the account has
+ * mReceiverSignatureRequired set to true, then all cryptocurrency transfers must be signed by this account's key, both
+ * for transfers in and out. If it is false, then only transfers out have to be signed by it. When the account is
+ * created, the payer account is charged enough Hbars so that the new account will not expire for the next
+ * mAutoRenewPeriod seconds. When it reaches the expiration time, the new account will then be automatically charged to
+ * renew for another mAutoRenewPeriod seconds. If it does not have enough Hbars to renew for that long, then the
+ * remaining Hbars are used to extend its expiration as long as possible. If it is has a zero balance when it expires,
+ * then it is deleted. This transaction must be signed by the payer account. If mReceiverSigRequired is false, then the
```

Comment:
mReceiverSigRequired seems to have some duplicate documentation in the first and second paragraphs